tools/slang-test/slang-test-main.cpp

```
@@ -1142,6 +1185,11 @@ static Result _executeRPC(
     outRes.standardError = exeRes.stdError;
     outRes.standardOutput = exeRes.stdOut;
     outRes.debugLayer = exeRes.debugLayer;
+    outRes.executionTimeMs = exeRes.executionTimeMs;
+
+    // Drain test-server stderr (phase timing diagnostics) before returning
+    // so it appears before the test result
+    context->drainTestServerStderr();
```

Shouldn't we report the error through
`outRes.standardError` rather than draining it?